frontend/src/components/elements/Tabs/Tabs.tsx

```diff
@@ -127,7 +127,6 @@ function Tabs(props: Props): ReactElement {
                   source={nodeLabel}
                   allowHTML={false}
                   isLabel
-                  isTabs
```

Comment:
Why was this removed?

Reply:
No longer using the `isTabs` prop -> previously used to restrict colored text in tabs, but product confirmed desire to allow it now.